.eslintrc.json

```diff
@@ -7,7 +7,14 @@
     "jest": true,
     "node": true
   },
-  "extends": ["eslint:recommended", "plugin:prettier/recommended"],
+  "plugins": ["node", "standard", "prettier"],
+  "extends": [
+    "eslint:recommended",
+    "standard",
+    "prettier",
+    "prettier/standard",
+    "plugin:prettier/recommended"
```

Comment:
Why not just run prettier --list-different?

Reply:
By connecting the prettier to eslint, we can automatically correct the errors of the prettier (through eslint).
or what did you mean?

Reply:
IMO eslint should not be used for formatting at all

Reply:
Why not? The project already uses `eslint --fix` and `prettier --write` (in lint-staged).
It turns out that the same files are being processed twice (by eslint and prettier).
You can process them in one pass, if you connect the prettier inside eslint.

In addition, if you use them separately, their rules may conflict.
Then in a text editor they constantly correct each other (if plugins are installed for eslint and prettier).